templates/amm.cpp.in

```diff
@@ -397,64 +396,71 @@ std::pair<double,double> vary_scale(const @ModelName@_mass_eigenstates& model, c
 
 } // anonymous namespace
 
-double @ModelName@_a_muon::calculate_a_muon(const @ModelName@_mass_eigenstates& model_, const softsusy::QedQcd& qedqcd)
+template <typename Lepton>
+double @ModelName@_amm::calculate_amm(const @ModelName@_mass_eigenstates& model_, const softsusy::QedQcd& qedqcd@extraIdxDecl@)
 {
    @ModelName@_mass_eigenstates model(model_);
 
-   VERBOSE_MSG("@ModelName@_a_muon: starting calculation of a_mu ...");
+   VERBOSE_MSG("@ModelName@_amm: starting calculation of a_mu ...");
 
    try {
       run_to_MSUSY(model);
       model.get_physical().clear();
    } catch (const Error& e) {
-      ERROR("@ModelName@_a_muon:" << e.what_detailed());
+      ERROR("@ModelName@_amm:" << e.what_detailed());
       return std::numeric_limits<double>::quiet_NaN();
    }
 
-   double m_muon_pole = muonPhysicalMass(qedqcd);
+   double m_muon_pole = leptonPhysicalMass<Lepton>(qedqcd@extraIdxUsage@);
```

Comment:
This variable should be renamed, maybe to `m_lepton_pole` or so.

Reply:
Done in 87ab68bf65f2b7047f0cd0fd08497fb23b2f508f.